### DISCUSSION TITLE
Is this project dead?

### DISCUSSION THREAD

**Discussion by geekyme-fsmk** [testsigmahq/testsigma#379] [Questions & Help (How to)]
We have an enterprise plan. Reached out several times on enterprise support. Dismal support quality. Looks like commits aren't rolling in the last few months. 

Is Testsigma abandonware right now? Worth an explanation to paying customers who aren't receiving the support they paid for.

**Comment by amitsavyasachi** [comment]
Hi @geekyme-fsmk ,

Apologies, I will check what went wrong and get back to you. Can you drop an email regarding the issue to support@testsigma.com and cc me (amit.savyasachi@testsigma.com).
